contingency/contingency-api/src/main/java/com/powsybl/contingency/json/ContingencyListDeserializer.java

```diff
@@ -28,29 +28,39 @@ public ContingencyList deserialize(JsonParser parser, DeserializationContext des
+                    }
+                    case "injectionCriterion" -> {
                         InjectionCriterionContingencyListDeserializer injectionCriterionContingencyListDeserializer = new InjectionCriterionContingencyListDeserializer();
                         return injectionCriterionContingencyListDeserializer.deserialize(parser, deserializationContext);
-                    case "hvdcCriterion":
+                    }
+                    case "hvdcCriterion" -> {
                         HvdcLineCriterionContingencyListDeserializer hvdcLineCriterionContingencyListDeserializer = new HvdcLineCriterionContingencyListDeserializer();
                         return hvdcLineCriterionContingencyListDeserializer.deserialize(parser, deserializationContext);
-                    case "lineCriterion":
+                    }
+                    case "lineCriterion" -> {
                         LineCriterionContingencyListDeserializer lineCriterionContingencyListDeserializer = new LineCriterionContingencyListDeserializer();
                         return lineCriterionContingencyListDeserializer.deserialize(parser, deserializationContext);
-                    case "twoWindingsTransformerCriterion":
+                    }
+                    case "twoWindingsTransformerCriterion" -> {
                         TwoWindingsTransformerCriterionContingencyListDeserializer twoWindingsTransformerCriterionContingencyListDeserializer = new TwoWindingsTransformerCriterionContingencyListDeserializer();
                         return twoWindingsTransformerCriterionContingencyListDeserializer.deserialize(parser, deserializationContext);
-                    case "threeWindingsTransformerCriterion":
+                    }
+                    case "threeWindingsTransformerCriterion" -> {
                         ThreeWindingsTransformerCriterionContingencyListDeserializer threeWindingsTransformerCriterionContingencyListDeserializer = new ThreeWindingsTransformerCriterionContingencyListDeserializer();
                         return threeWindingsTransformerCriterionContingencyListDeserializer.deserialize(parser, deserializationContext);
-                    case "list":
+                    }
+                    case "list" -> {
                         ListOfContingencyListsDeserializer listOfContingencyListsDeserializer = new ListOfContingencyListsDeserializer();
                         return listOfContingencyListsDeserializer.deserialize(parser, deserializationContext);
-                    default:
-                        throw new IllegalStateException("Unexpected field: " + parser.getCurrentName());
+                    }
+                    case "identifier" -> {
+                        IdentifierContingencyListDeserializer identifierContingencyListDeserializer = new IdentifierContingencyListDeserializer();
+                        return identifierContingencyListDeserializer.deserialize(parser, deserializationContext);
```

Comment:
here is the main change

Reply:
Ok, I read your PR to quickly. The attribute was already serialized, but its deserialization was not managed.

Could you add a unit test serializing and deserializing a `ContingencyList` with identifiers?